nightwatch/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ENV=$1
+[ -z "$ENV" ] && ENV='default'
+
+if [ "$ENV" != "default" ] ; then
```

Comment:
i'd say just load the environment variables always if they exist
you can also do:
```shell
 set -o allexport
 source ../mycredentials
 set +o allexport
```